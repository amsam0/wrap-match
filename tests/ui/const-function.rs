fn main() {}

#[wrap_match::wrap_match]
const fn const_function() -> Result<(), ()> {
    Ok(())
}
