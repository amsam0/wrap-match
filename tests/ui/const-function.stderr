error: wrap_match cannot be used on const functions because the log crate cannot be used in const contexts
 --> tests/ui/const-function.rs:4:1
  |
4 | const fn const_function() -> Result<(), ()> {
  | ^^^^^
